src/mixins/FetchFacesMixin.js

```diff
@@ -122,7 +122,7 @@ export default {
 
 				fetchedFiles = fetchedFiles
 					.map(file => genFileInfo(file))
-					.map(file => ({ ...file, filename: file.realpath.replace(`/${getCurrentUser().uid}/files`, '') }))
+					.map(file => ({ ...file, filename: file.realpath.replace(`/${getCurrentUser().uid}/files`, `/files/${getCurrentUser().uid}`) }))
```

Comment:
It breaks currently because the filename doesn't resolve to a valid DAV path. With this change user user files are resolved to valid DAV paths allowing the favorite action to work.